src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -133,20 +101,14 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
 
 
         [Test]
-        public void Setter_dependencies_should_be_supported()
+        public void Setter_injection_should_be_enabled_by_default()
         {
             using (var builder = TestContainerBuilder.ConstructBuilder())
             {
                 builder.Configure(typeof(SomeClass), DependencyLifecycle.InstancePerCall);
                 builder.Configure(typeof(ClassWithSetterDependencies), DependencyLifecycle.SingleInstance);
-                builder.ConfigureProperty(typeof(ClassWithSetterDependencies), "EnumDependency", SomeEnum.X);
-                builder.ConfigureProperty(typeof(ClassWithSetterDependencies), "SimpleDependency", 1);
-                builder.ConfigureProperty(typeof(ClassWithSetterDependencies), "StringDependency", "Test");
-
-                var component = (ClassWithSetterDependencies) builder.Build(typeof(ClassWithSetterDependencies));
-                Assert.AreEqual(component.EnumDependency, SomeEnum.X);
-                Assert.AreEqual(component.SimpleDependency, 1);
-                Assert.AreEqual(component.StringDependency, "Test");
+
+                var component = (ClassWithSetterDependencies)builder.Build(typeof(ClassWithSetterDependencies));
```

Comment:
I think @timbussmann is spot on. We should rename it to `TestContainerBuilder.Construct()` or `TestContainerBuilder.ConstructContainer()` and also rename the `var builder` to `var container`. But then in all the tests